README.md

```diff
@@ -18,72 +19,132 @@ npm run-script build
 To generate a docker container, one may issue the following command:
 
 ```shell
+# you may need sudo on your machine:
+# https://docs.docker.com/engine/installation/linux/linux-postinstall/
 docker build -t <tag> -f docker/Dockerfile .
 ```
 
-Then an image tagged as `<tag>` will be made available. Do notice that a pre-built "official" version for this component may be found at dojot's [dockerhub](https://hub.docker.com/r/dojot/data-broker/).
+Then an image tagged as `<tag>` will be made available. Do notice that a
+pre-built "official" version for this component may be found at dojot's
+[dockerhub](https://hub.docker.com/r/dojot/data-broker/).
 
-## How to run
+## Configuration
 
-Once built:
+In order to properly start data-broker, the following variables can be set. All
+values are the default ones.
 
 ```shell
-npm run subscription
+#
+# Data-broker cache configuration
+#
+
+# Type of cache used. Currently only Redis is suported.
+# If set to 'NOCACHE' auth will work without cache.
+# Disabling cache usage considerably degrades performance.
+export DATABROKER_CACHE_NAME="redis"
+# username to access the cache database
+export DATABROKER_CACHE_USER="redis"
+# password to acces the cache database
+export DATABROKER_CACHE_PWD=""
+# ip or hostname where the cache can be found
+export DATABROKER_CACHE_ADDRESS="data-broker-redis"
+# Redis port
+export DATABROKER_CACHE_PORT=6379
+# cach database name (or number)
+export DATABROKER_CACHE_DATABASE="0"
+
+#
+# Data-broker main config.
+#
+
+# Port used by data-broker to receive request
+export DATABROKER_PORT="80"
+
+#
+# Other services
+#
+# Kafka address
+export KAFKA_ADDRESS="kafka"
```

Comment:
These variables will change a bit when updating to the newest version of dojot-module.